salt/lib/menu.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+##############################################################
+# Copyright (c) 2019 Saltstack formulas, The OpenSDS Authors
```

Comment:
Is this code copied from any other open source? (is it standard saltstack formula file?) -- please confirm.
If its is copied, based on the licensing, we need to handle. Need to add the right comments about why and how we are using.
Also, we must check whether it is ok to add OpenSDS Authors, if we have copied as is.

Reply:
Hi @skdwriting 
Thanks for reviewing - I have fixed the copyright in `salter.sh`.

I removed this `menu.py` file because its not needed or documented.

Reply:
+1